test/toolshed/date_test.exs

```diff
@@ -0,0 +1,19 @@
+defmodule Toolshed.DateTest do
+  use ExUnit.Case
+
+  test "date/0 returns current date in unix format" do
+    # There's a race condition on the time that's returned between the
+    # two date functions if we catch the seconds changing.
+
+    expected1 = unix_date_output()
+    actual = Toolshed.Date.main()
```

Comment:
```suggestion
    actual = Toolshed.date()
```
This will require an update to `toolshed.ex` to delegate `date/0` to `Toolshed.Date.main/0`. I think this is the right way to go since users should never be calling `Toolshed.Date.main` and if the test calls `Toolshed.date` then we'll know that it's properly exported.

Reply:
The suggestions are accommodated now, thanks